packages/app/src/messages/components/templates-select-modal/templates-select-modal.component.tsx

```diff
@@ -0,0 +1,57 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+
+import React from "react"
+import { defineMessages } from "react-intl"
+import { intl } from "App/__deprecated__/renderer/utils/intl"
+import { FunctionComponent } from "App/__deprecated__/renderer/types/function-component.interface"
+import { ModalSize } from "App/__deprecated__/renderer/components/core/modal/modal.interface"
+import { ModalDialog } from "App/ui/components/modal-dialog"
+import Table, {
+  Col,
+  Row,
+  RowSize,
+} from "App/__deprecated__/renderer/components/core/table/table.component"
+import { Template } from "App/templates/dto"
+import { TemplatesSelectModalTestIds } from "App/messages/components/templates-select-modal/templates-select-modal-test-ids.enum"
+export interface TemplatesSelectModalProps {
```

Comment:
Let's move it to separate file

Reply:
Props moved in [75ca6c9](https://github.com/mudita/mudita-center/pull/823/commits/75ca6c9f6535fba4f0a5872a3c814e643025e196)